beacon_node/beacon_chain/src/kzg_utils.rs

```diff
@@ -30,19 +30,6 @@ pub fn validate_blobs<T: EthSpec>(
     blobs: &[Blob<T>],
     kzg_proofs: &[KzgProof],
 ) -> Result<bool, KzgError> {
-    // TODO(sean) batch verification fails with a single element, it's unclear to me why
```

Comment:
Was this not actually an issue?

Reply:
Didn't see a failure on running with this on devnet 7. I'm more inclined to catch this behaviour on devnets and fix it assuming it's still happening.